recipes/methpype/meta.yaml

```
@@ -0,0 +1,39 @@
+package:
+  name: methpype
+  version: 1.0.1
+
+source:
+  url: https://files.pythonhosted.org/packages/00/88/eab68b60ad237ba0f9b69f862b6b92553a9646b4e5620a06e58803fa645d/methpype-1.0.1.tar.gz
+  sha256: acf192a02388a2eb71fe4ae6d0bac13b174c53fb3bb98f95eaaa9e17b2394597
+
+build:  
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Could you please add ``noarch: python``?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```